assets/src/dashboard/components/cardGrid/index.js

```diff
@@ -47,15 +50,24 @@ const DashboardGrid = styled.div(
   ${theme.breakpoint.min} {
     grid-column-gap: ${theme.grid.columnGap.min}px;
   }
+  
+  ${KEYBOARD_USER_SELECTOR} &:focus {
+    outline: 2px solid ${rgba(theme.colors.bluePrimary, 0.85)};
+  }
 `
 );
 DashboardGrid.propTypes = {
   columnHeight: PropTypes.number.isRequired,
   columnWidth: PropTypes.number.isRequired,
 };
 
-const CardGrid = ({ children, pageSize }) => (
+const CardGrid = ({ ariaLabel, children, pageSize, gridRef }) => (
```

Comment:
We can make the prop just `ref` using `forwardRef` here. That way we don't need a custom prop name.

https://reactjs.org/docs/forwarding-refs.html

Reply:
I agree with Carlos, but it doesn't seem like `gridRef` was being used in the parent (I could be wrong) and we could just create it here

Reply:
great idea, thanks!